package.json

```diff
@@ -1,6 +1,6 @@
 {
   "dependencies": {
-    "@abds/components": "rc",
```

Comment:
Why not `latest` if not RC? we are suppose to be doing an RC to test our apps before we release 😅

Reply:
I think it would be good if we just set it to the version we expect to be used. This prevents our apps from being broken in the case users will be viewing it. I think if we want to use rc it is fine, but I think it should probably only be used locally for testing.

Reply:
yeah, I guess that makes sense, we would just want to make sure that we update this version when we come into the app to update with new components/content from new releases.

Is there documentation on what to do when updating the example apps?

Reply:
I think I misunderstood the comment. I will update to use `latest`.